README.md

```
@@ -1,58 +1,312 @@
+
+- **Scanner Interface**  
+  URL: [https://dhakametrorail.bsite.net/System/Scanner](https://dhakametrorail.bsite.net/System/Scanner)  
+  This interface is used by the metro station system to scan and validate QR code tickets in real time.
+
+## 📁 Code Structure
+This project follows **Clean Architecture** principles and incorporates **industry best practices** to ensure modularity, testability, and maintainability.
+
+### 🔧 Clean Architecture
+
+<p align="center">
+  <img src="docs/Images/clean-architecture-.NET-core.jpg" alt="Clean Architecture" width="320" />
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: img

(MD033, no-inline-html)